xlators/cluster/dht/src/dht-common.c

```diff
@@ -10884,7 +10884,7 @@ dht_notify(xlator_t *this, int event, void *data, ...)
                 gf_msg(this->name, GF_LOG_WARNING, 0, DHT_MSG_CHILD_DOWN,
                        "Received CHILD_DOWN. Exiting");
                 if (conf->defrag) {
-                    gf_defrag_stop(conf, GF_DEFRAG_STATUS_FAILED, NULL);
+                    gf_defrag_stop(conf, GF_DEFRAG_STATUS_FAILED, NULL, true);
```

Comment:
We should use `_gf_true_` instead of `true`. I'm not sure if this is necessary to support older compilers, or it's just an obsolete coding convention; nevertheless, that is the current convention, so please adhere to that.

Reply:
Done